modules/core/arrow-core-extensions/src/main/kotlin/arrow/core/extensions/either.kt

```diff
@@ -149,6 +151,15 @@ interface EitherFoldable<L> : Foldable<EitherPartialOf<L>> {
     fix().foldRight(lb, f)
 }
 
+@extension
+interface EitherBifoldable : Bifoldable<ForEither> {
+  override fun <A, B, C> EitherOf<A, B>.bifoldLeft(c: C, f: (C, A) -> C, g: (C, B) -> C): C =
+    fix().fold({ f(c, it) }, { g(c, it) })
```

Comment:
These implementations should delegate to methods in `Either` so thatthe data type also includes `bifoldLeft` and `bifoldRight`

Reply:
ok, but what about Validated and Tuple, should I move for them too?